### PR TITLE
Implement Logging Structure

### PR DESCRIPTION
**Pull request by VirajSalaka**
### Purpose

This includes the basic structure for logging implementation. Logfile parsing happens within adapter implementation. 

### Issues

Fixes #

### Automation tests
 - Unit tests added: No
 - Integration tests added: No

### Tested environments

Not Tested

---
#### Maintainers: Check before merge
- [ ] Assigned 'Type' label
- [ ] Assigned the project
- [ ] Validated respective github issues
- [ ] Assigned milestone to the github issue(s)

### CODE REVIEW

**Comment by VirajSalaka** [comment]
The `VirajSalaka` fork can be removed from go.mod file once the following PR is merged.
 https://github.com/wso2/product-microgateway/pull/2076.

File: resources/conf/log_config.toml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# The logging configuration file for global adapter
+######### root Level ########
+logfile = "logs/adapter.log"
+logLevel = "INFO"
+
+[rotation]
+MaxSize = 10    #meabytes
```
Comment (mpmunasinghe) [inline resources/conf/log_config.toml:7]: Typo #meabytes shiuld be #megabytes

File: internal/logger/logger.go [hunk 2]
```diff
@@ -0,0 +1,51 @@
+/*
+ *  Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (mpmunasinghe) [inline internal/logger/logger.go:2]: With new guidelines given we can omit "All Rights Reserved" from the header as we later provide rights through apache licensing.

**Review by mpmunasinghe — APPROVED** [review]
Consider fixing the minor comments provided in a later update.

File: internal/logger/logger.go (hunk 2 above)
Comment (VirajSalaka) [reply internal/logger/logger.go]: fixed with https://github.com/wso2-enterprise/choreo-connect-global-adapter/pull/3/commits/f26916c5ef99e0b5f9b9ff19f08f600721f4c516

File: resources/conf/log_config.toml (hunk 1 above)
Comment (VirajSalaka) [reply resources/conf/log_config.toml]: fixed with https://github.com/wso2-enterprise/choreo-connect-global-adapter/pull/3/commits/f26916c5ef99e0b5f9b9ff19f08f600721f4c516

**Pull request merged**